### ISSUE TITLE
[pfto] ops not normalized in traced graph of pfto

### ISSUE THREAD

**Issue by take-cheeze** [pfnet/pytorch-pfn-extras#673]
ops like `torch.clip`  won't be treated as `torch.clamp` without it and will end up symbolic function not found
- Add tests with torch.clip
- Make it apply normalize ops pass https://sourcegraph.com/github.com/pytorch/pytorch@8ee59280d78a4fefc4de0da04b287e067c28de0d/-/blob/torch/csrc/jit/passes/normalize_ops.cpp?L166:6
